src/components/Alert/Alert.styled.ts

```diff
@@ -1,3 +1,59 @@
-import styled from "styled-components";
+import styled, { css, keyframes } from "styled-components";
 
-export const StyledAlert = styled.div``;
+const appear = keyframes`
+  from {
+    transform: translateY(100%);
+  }
+
+  to {
+    transform: translateY(0%);
+  }
+`;
+
+export const StyledAlert = styled.div(({ color }) => [
+  css`
+    border-radius: 4px;
+    margin: 0.5rem;
+    padding: 1rem;
+    position: relative;
+    font-size: 1.2rem;
+    color: white;
+    display: inline-flex;
+    align-items: center;
+    width: 25rem;
+    gap: 0.5rem;
```

Comment:
Suggestion: better to use px for values less than 1 rem to maintain the constant visibility while changing screen sizes